lefthook.yml

```diff
@@ -0,0 +1,10 @@
+lefthook: |
+  cd webapp
+  yarn lefthook
+source_dir: webapp/packages/core-cli
+output: false
+pre-commit:
+  parallel: true
+  scripts:
+    "license-check.mjs":
+      runner: node
```

Comment:
it's better to expose `license-check.mjs` as a bin command from `core-cli` and run it like: `yarn core-cli-check-license`

also, that's strange that you specified '.mjs' while `core-cli`'s type is "module"